### PR TITLE
W-14722930-Remove-Flow-Designer-articles-from-Prod-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
W-14722930-Remove-Flow-Designer-articles-from-Prod-CP

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .xref:index.adoc[Design Center]
-* xref:index.adoc[About Design Center]
+* xref:index.adoc[Overview]
```
Comment (hannanelson) [inline modules/ROOT/nav.adoc]:
```suggestion
* xref:index.adoc[Design Center Overview]
```

Comment (hannanelson) [reply modules/ROOT/nav.adoc]: You could remove the hard-coded link text from the nav.adoc, if the title matches the page title. It saves maintenance if you change the title down the road.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -3,17 +3,9 @@ ifndef::env-site,env-github[]
...
...wilio.adoc, fd-elements-of-flows.adoc, import-template.adoc, to-create-a-mule-application-project.adoc, fd-connections.adoc, fd-connection-create.adoc, fd-connection-switch-in-project.adoc, fd-connection-edit.adoc, fd-connection-delete.adoc, fd-connection-view-audit-logs.adoc, about-data-types.adoc, fd-supported-simple-data-types.adoc, fd-custom-data-types.adoc, fd-integrating-systems.adoc, dynamic-expression-field.adoc, custom-expression-field.adoc, fd-add-core-component.adoc, fd-iterating.adoc, fd-scatter-gather.adoc, reference-flow-task-design-center.adoc, error-handling-task-design-center.adoc, fd-transform.adoc, fd-mapping-examples.adoc, fd-tour-transform-card.adoc, get-mapping-recommendations.adoc, fd-store-data-top.adoc, to-create-and-populate-a-variable.adoc, fd-store-reuse-output.adoc, manage-dependency-versions-design-center.adoc, fd-share-to-studio.adoc, jump-runtime-manager-task.adoc, fd-project-history.adoc, fd-troubleshooting.adoc, logger-task-design-center.adoc, troubleshooting-reference.adoc, viewing-problems-fd-canvas.adoc, view-clear-logs-task.adoc, fd-deploy.adoc, promote-app-prod-env-design-center.adoc, fd-deploy-app-from-rm.adoc, fd-publish-app-as-template.adoc
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc:6]: If you include all these page aliases, you might want to include a note that indicates Flow Designer was EOLed and maybe include a link to the archive site (kind of like what you did with the Studio RN landing page).

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply modules/ROOT/pages/index.adoc:6]: OK! Thanks!

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -3,17 +3,11 @@ ifndef::env-site,env-github[]
...
+Anypoint Design Center is a development environment that includes xref:design-create-publish-api-specs.adoc[API Designer], a tool that enables you to create API specifications in several modeling languages and create RAML API fragments. After you create a specification or fragment, publish it to Anypoint Exchange for anyone in your MuleSoft organization to use.
 
-* xref:design-create-publish-api-specs.adoc[API Designer]
-+
-API Designer enables you to create API specifications in several modeling languages and create RAML API fragments. After you create a specification or fragment, publish it to Anypoint Exchange for anyone in your MuleSoft organization to use.
-
-* xref:about-designing-a-mule-application.adoc[Flow Designer]
-+
-In Flow Designer, you create Mule applications to integrate systems into workflows.
+NOTE: Flow Designer is https://www.mulesoft.com/legal/versioning-back-support-policy[end of life] and its documentation is now located at https://archive.docs.mulesoft.com/design-center/index[Flow Designer archive site].
```
Comment (hannanelson) [inline modules/ROOT/pages/index.adoc]: Should the URL be to https://help.mulesoft.com/s/article/MuleSoft-Product-Feature-Retirements?

Comment (Crispy-Salesforce) [reply modules/ROOT/pages/index.adoc]: Great catch! Thanks!

**Review by hannanelson — APPROVED** [review]

**Pull request merged**